shared/modules/types.go

```diff
@@ -1,6 +1,228 @@
+	GetMessagePauseAppFee() string
+	GetMessageUnpauseAppFee() string
+	GetMessageStakeValidatorFee() string
+	GetMessageEditStakeValidatorFee() string
+	GetMessageUnstakeValidatorFee() string
+	GetMessagePauseValidatorFee() string
+	GetMessageUnpauseValidatorFee() string
+	GetMessageStakeServiceNodeFee() string
+	GetMessageEditStakeServiceNodeFee() string
+	GetMessageUnstakeServiceNodeFee() string
+	GetMessagePauseServiceNodeFee() string
+	GetMessageUnpauseServiceNodeFee() string
+	GetMessageChangeParameterFee() string
+
+	// ACL Queries
+	GetAclOwner() string
+	GetBlocksPerSessionOwner() string
+	GetAppMaxChainsOwner() string
+	GetAppMinimumStakeOwner() string
+	GetAppBaselineStakeRateOwner() string
+	GetAppStakingAdjustmentOwner() string
+	GetAppUnstakingBlocksOwner() string
+	GetAppMinimumPauseBlocksOwner() string
+	GetAppMaxPausedBlocksOwner() string
+	GetServiceNodeMinimumStakeOwner() string
+	GetServiceNodeMaxChainsOwner() string
+	GetServiceNodeUnstakingBlocksOwner() string
+	GetServiceNodeMinimumPauseBlocksOwner() string
+	GetServiceNodeMaxPausedBlocksOwner() string
+	GetFishermanMinimumStakeOwner() string
+	GetFishermanMaxChainsOwner() string
+	GetFishermanUnstakingBlocksOwner() string
+	GetFishermanMinimumPauseBlocksOwner() string
+	GetFishermanMaxPausedBlocksOwner() string
+	GetValidatorMinimumStakeOwner() string
+	GetValidatorUnstakingBlocksOwner() string
+	GetValidatorMinimumPauseBlocksOwner() string
+	GetValidatorMaxPausedBlocksOwner() string
+	GetValidatorMaximumMissedBlocksOwner() string
+	GetProposerPercentageOfFeesOwner() string
+	GetValidatorMaxEvidenceAgeInBlocksOwner() string
+	GetMissedBlocksBurnPercentageOwner() string
+	GetDoubleSignBurnPercentageOwner() string
+	GetServiceNodesPerSessionOwner() string
+	GetMessageDoubleSignFeeOwner() string
+	GetMessageSendFeeOwner() string
+	GetMessageStakeFishermanFeeOwner() string
+	GetMessageEditStakeFishermanFeeOwner() string
+	GetMessageUnstakeFishermanFeeOwner() string
+	GetMessagePauseFishermanFeeOwner() string
+	GetMessageUnpauseFishermanFeeOwner() string
+	GetMessageFishermanPauseServiceNodeFeeOwner() string
+	GetMessageTestScoreFeeOwner() string
+	GetMessageProveTestScoreFeeOwner() string
+	GetMessageStakeAppFeeOwner() string
+	GetMessageEditStakeAppFeeOwner() string
+	GetMessageUnstakeAppFeeOwner() string
+	GetMessagePauseAppFeeOwner() string
+	GetMessageUnpauseAppFeeOwner() string
+	GetMessageStakeValidatorFeeOwner() string
+	GetMessageEditStakeValidatorFeeOwner() string
+	GetMessageUnstakeValidatorFeeOwner() string
+	GetMessagePauseValidatorFeeOwner() string
+	GetMessageUnpauseValidatorFeeOwner() string
+	GetMessageStakeServiceNodeFeeOwner() string
+	GetMessageEditStakeServiceNodeFeeOwner() string
+	GetMessageUnstakeServiceNodeFeeOwner() string
+	GetMessagePauseServiceNodeFeeOwner() string
+	GetMessageUnpauseServiceNodeFeeOwner() string
+	GetMessageChangeParameterFeeOwner() string
+}
+
+// TODO (Team) move to use proto string() and deprecate #147
```

Comment:
Why are we deprecating the state hash computation?

Reply:
What? These are `param name` shared constants that should not live in `shared`

Reply:
I'm confused as well. Do you want me to do this as part of https://github.com/pokt-network/pocket/issues/147?

Reply:
I'm fine for you to do it as long as you understand that it's just about removing these shared variables and utilizing the `protobuf.String()` function so that the naming of the parameters may be defined by the language agnostic generator